sdk/core/core-sse/review/core-sse.api.md

```diff
@@ -0,0 +1,23 @@
+## API Report File for "@azure/core-sse"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+// @public
+export interface EventMessage {
+    data: string;
+    event: string;
+    id: string;
+    retry?: number;
+}
+
+// @public
+export function toSSE(chunkIter: ReadableStream<Uint8Array>): AsyncIterable<EventMessage>;
```

Comment:
name nit, can we call this something like `iterateSseStream` or something to indicate what's going on? I imagine in the future we might want to support preserving the events as a stream directly (but still JSON.parsing them and whatnot)